.vscode/settings.json

```diff
@@ -2,13 +2,13 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "files.eol": "\n",
   "editor.tabSize": 2,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
```

Comment:
Prettier was already in there, so I figured it was ok to add this, I was sick of not having it auto-format while making changes.